README.md

```diff
@@ -194,6 +194,28 @@ This will create a file `ccm-linode.yaml` which you can use to deploy the CCM.
 
 Note: Your kubelets, controller-manager, and apiserver must be started with `--cloud-provider=external` as noted in the following documentation.
 
+## Deployment Through Helm Chart
+
+Use the helm chart located under './deploy/chart'. This dir has the manifest for Linode CCM. There are two arguments required.
+
+The first argument must be a Linode APIv4 Personal Access Token with all permissions.
+(https://cloud.linode.com/profile/tokens)
+
+The second argument must be a Linode region.
+(https://api.linode.com/v4/regions)
+
+To deploy CCM run the following helm command:
+```sh
+helm install linode-ccm ./deploy/chart --set apiToken=$LINODE_API_TOKEN,region=$REGION
```

Comment:
for completeness of the install steps can we add a `git clone` and `cd` command in preparation of using the helm install command

Reply:
To deploy CCM run the following helm command once you are in the ccm root dir:
```sh
git clone SSH_LINK

cd /Path/To/linode-cloud-controller-manager

helm install linode-ccm ./deploy/chart --set apiToken=$LINODE_API_TOKEN,region=$REGION
```

Does this look better?

Reply:
I would say use full actual SSH_LINK value, and then just do ssomething like  cd ./linode-cloud-controller-manager

Reply:
Yup did that. Added the changes in the new commit